vector/src/main/res/values/strings.xml

```diff
@@ -2852,6 +2852,8 @@
     <string name="labs_auto_report_uisi_desc">Your system will automatically send logs when an unable to decrypt error occurs</string>
     <string name="labs_enable_thread_messages">Enable Thread Messages</string>
     <string name="labs_enable_thread_messages_desc">Note: app will be restarted</string>
+    <string name="labs_show_latest_profile">Show latest user info</string>
+    <string name="labs_show_latest_profile_description">Use the latest profile info (avatar and display name) for all the messages.</string>
```

Comment:
For coherency, we should use the same verb for the title and the description. `Use` or `Show`. But I am not sure what is the best choice.
Keys will have to be renamed too.

Reply:
Imo `Show` is more specific and thus my personal preference